2022-10-28 00:19:24.180 -05:00 [ERR] An unhandled exception has occurred while executing the request
System.ArgumentNullException: Value cannot be null.
Parameter name: dbContextTransaction
   at Microsoft.EntityFrameworkCore.Utilities.Check.NotNull[T](T value, String parameterName)
   at Microsoft.EntityFrameworkCore.Storage.DbContextTransactionExtensions.GetDbTransaction(IDbContextTransaction dbContextTransaction)
   at Release.Helper.Data.Core.BaseUnitOfWork.Rollback()
   at Prod.GESTORPLUS.Servicios.Applicacion.RegisterAplicacion.Registrar(Register register) in C:\gestorplus\Prod.GESTORPLUS.Core\Applicacion\RegisterAplicacion.cs:line 66
   at Prod.GESTORPLUS.Servicios.Controllers.RegisterComandoController.Registrar(Register register) in C:\gestorplus\Prod.GESTORPLUS.Core\Controllers\Comandos\RegisterComandoController.cs:line 25
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2022-10-28 00:20:17.681 -05:00 [ERR] An unhandled exception has occurred while executing the request
System.ArgumentNullException: Value cannot be null.
Parameter name: dbContextTransaction
   at Microsoft.EntityFrameworkCore.Utilities.Check.NotNull[T](T value, String parameterName)
   at Microsoft.EntityFrameworkCore.Storage.DbContextTransactionExtensions.GetDbTransaction(IDbContextTransaction dbContextTransaction)
   at Release.Helper.Data.Core.BaseUnitOfWork.Rollback()
   at Prod.GESTORPLUS.Servicios.Applicacion.RegisterAplicacion.Registrar(Register register) in C:\gestorplus\Prod.GESTORPLUS.Core\Applicacion\RegisterAplicacion.cs:line 66
   at Prod.GESTORPLUS.Servicios.Controllers.RegisterComandoController.Registrar(Register register) in C:\gestorplus\Prod.GESTORPLUS.Core\Controllers\Comandos\RegisterComandoController.cs:line 25
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2022-10-28 00:22:10.863 -05:00 [ERR] An unhandled exception has occurred while executing the request
System.ArgumentNullException: Value cannot be null.
Parameter name: dbContextTransaction
   at Microsoft.EntityFrameworkCore.Utilities.Check.NotNull[T](T value, String parameterName)
   at Microsoft.EntityFrameworkCore.Storage.DbContextTransactionExtensions.GetDbTransaction(IDbContextTransaction dbContextTransaction)
   at Release.Helper.Data.Core.BaseUnitOfWork.Rollback()
   at Prod.GESTORPLUS.Servicios.Applicacion.RegisterAplicacion.Registrar(Register register) in C:\gestorplus\Prod.GESTORPLUS.Core\Applicacion\RegisterAplicacion.cs:line 66
   at Prod.GESTORPLUS.Servicios.Controllers.RegisterComandoController.Registrar(Register register) in C:\gestorplus\Prod.GESTORPLUS.Core\Controllers\Comandos\RegisterComandoController.cs:line 25
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
